packages/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/SnapshotUtils.h

```diff
@@ -0,0 +1,159 @@
+#pragma once
+
+#include "LayoutAnimationsManager.h"
+#include "PropsRegistry.h"
+
+#include <react/renderer/mounting/MountingOverrideDelegate.h>
+#include <react/renderer/mounting/ShadowView.h>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+#ifdef __ANDROID__ // parse color on android
+#include <iomanip>
+#include <sstream>
+#endif
+
+namespace reanimated {
+
+#ifdef __ANDROID__
```

Comment:
Could we use a const expression here instead of `#ifdef`? This approach is more type-safe and doesn't have any performance overhead also

Reply:
Unfortunately `constexpr` can't be used with strings. I was trying several workarounds and each of them required many typecasts and was generating lot's of code.